app/src/main/java/org/jboss/hal/client/bootstrap/tasks/ReadEnvironment.java

```diff
@@ -140,6 +144,9 @@ public Completable call(FlowContext context) {
                     }
                     user.setAuthenticated(true);
                     logger.debug("User info: {} {}", user.getName(), user.getRoles());
+
+                    ModelNode step = result.step(2).get(RESULT);
+                    environment.setPatchingEnabled(step.get("patching").isDefined());
```

Comment:
Hard coded string could be replaced by `ModelDescriptionConstants.PATCHING`.

Reply:
Please replace with `ModelDescriptionConstants.PATCHING`

Reply:
changed, thanks!